From cdef64baccfe5525accdeebdbde1cb927819be57 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:02:24 -0400
Subject: [usb] xhci: remove unused 'ep_ring' variable in handle_cmd_completion()

Message-id: <1410887463-20674-73-git-send-email-dzickus@redhat.com>
Patchwork-id: 95660
O-Subject: [RHEL7 PATCH 072/591] xhci: remove unused 'ep_ring' variable in handle_cmd_completion()
Bugzilla: 1032371
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1032371

commit fd54498733f8c372deca99892ae8cae0799dfe68
Author: Xenia Ragiadakou <burzalodowa@gmail.com>
Date:   Mon Sep 9 13:29:54 2013 +0300

    xhci: remove unused 'ep_ring' variable in handle_cmd_completion()

    This patch removes the variable 'ep_ring' that is assigned in
    TRB_CONFIG_EP switch case but never used.

    Signed-off-by: Xenia Ragiadakou <burzalodowa@gmail.com>
    Signed-off-by: Sarah Sharp <sarah.a.sharp@linux.intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/xhci-ring.c b/drivers/usb/host/xhci-ring.c
index 481ee04..7e55faf 100644
--- a/drivers/usb/host/xhci-ring.c
+++ b/drivers/usb/host/xhci-ring.c
@@ -1462,7 +1462,6 @@ static void handle_cmd_completion(struct xhci_hcd *xhci,
  struct xhci_input_control_ctx *ctrl_ctx;
  struct xhci_virt_device *virt_dev;
  unsigned int ep_index;
- struct xhci_ring *ep_ring;
  unsigned int ep_state;
 
  cmd_dma = le64_to_cpu(event->cmd_trb);
@@ -1542,7 +1541,6 @@ static void handle_cmd_completion(struct xhci_hcd *xhci,
     ep_index != (unsigned int) -1 &&
       le32_to_cpu(ctrl_ctx->add_flags) - SLOT_FLAG ==
       le32_to_cpu(ctrl_ctx->drop_flags)) {
-   ep_ring = xhci->devs[slot_id]->eps[ep_index].ring;
    ep_state = xhci->devs[slot_id]->eps[ep_index].ep_state;
    if (!(ep_state & EP_HALTED))
     goto bandwidth_change;
-- 
1.7.1